flowmachine/flowmachine/features/subscriber/unique_subscribers_from_queries.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Given a list of queries with a 'subscriber' column, returns a 1-column table of unique subscribers.
+"""
+
+from typing import List
+
+from flowmachine.core.query import Query
+
+
+class UniqueSubscribersFromQueries(Query):
+    """
+    Class representing unique subscribers across a set of queries
+
+    Parameters
+    ----------
+    query_list: List[Query]
+        A list of Query objects, each of which must include a `subscriber` column
+
+    """
+
+    # Review question: Does this need to be 'subscriber' or can we make the column to select a parameter?
+
+    def __init__(self, query_list: List[Query]):
```

Comment:
Is it worth making the params for this `(self, query_list, column)`and letting it just return unique values in whatever we pass it?

Reply:
Sounds reasonable to me (but change the name to UniqueFromQueries or similar)

Reply:
Yes, I think that's a good idea. We could even accept a list of `columns` - that way we could use the query to get the set of all distinct (subscriber, location) pairs from several queries, for example.

I'd suggest we should move this to 'flowmachine/features/utilities', since it's not specifically subscriber-related. And perhaps `UniqueValuesFromQueries` would be an appropriate name?

There's a separate question of how we want to _expose_ this through the API - we may decide that for now it will only be exposed with `column` hard-coded to `"subscriber"`, as an allowed input to other queries' `subscriber_subset` parameters. But that's a separate question, and can be handled in a separate PR.

Reply:
Name changed, params updated

Reply:
Thanks! Could you move it into 'flowmachine/features/utilities' as well, please?